app/controllers/api/tasks_controller.rb

```diff
@@ -59,26 +59,26 @@ def mark_active
       return render 'bad_request', status: :bad_request if not @task.update(completed: false)
       render 'show', status: :ok
     end
-  end
```

Comment:
you closed the `class` with an `end` too early and the private methods is not defined inside the `class` therefore the `before_action` method couldn't access the private methods below.